apps/site/src/app/admin/applicants/components/ApplicantReviewerIndicator.tsx

```diff
@@ -0,0 +1,46 @@
+import Box from "@cloudscape-design/components/box";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+import StatusIndicator from "@cloudscape-design/components/status-indicator";
+
+interface IndicatorContainerProps {
+	number: number;
+	hasReviewed: boolean;
+}
+
+const IndicatorContainer = ({
+	number,
+	hasReviewed,
+}: IndicatorContainerProps) => {
+	return (
+		<>
+			<Box variant="awsui-key-label">Reviewer {number}</Box>
+			{hasReviewed ? (
+				<StatusIndicator>Reviewed</StatusIndicator>
+			) : (
+				<StatusIndicator type="pending">Not Reviewed</StatusIndicator>
+			)}
```

Comment:
Note [nice to have]: next to "Reviewer 1" and "Reviewer 2," I think it will be helpful to show the UID of the reviewer next to them

Reply:
Note: just pointing out for context that in previous years we omitted who submitted the review (unless it was yourself) to reduce potential inter-reviewer bias. Ultimately, up to you all to decide what to show.

Reply:
Don't think log/applications have decided if and how to assign reviewers, but if they decide to keep track of who's assigned to which app, then I think showing who submitted reviews would be fine because reviewers could still see who their "co-reviewer" would be. Showing reviewers would also help everyone keep track of all the apps, in my opinion.

Reply:
Follow up to this, it seems that log/applications will be deciding how to assign apps to reviewers in their next meeting in a few days, so we can keep this. I'll keep everyone updated in case something happens.